ember_debug/view-debug.js

```diff
@@ -264,18 +264,20 @@ export default EmberObject.extend(PortMixin, {
     this.sendMessage('stopInspecting', {});
   },
 
+  send() {
+    if (this.isDestroying) {
+      return;
+    }
+    this.releaseCurrentObjects();
+    let tree = this.viewTree();
+    if (tree) {
+      this.sendMessage('viewTree', { tree });
+    }
+  },
+
   scheduledSendTree() {
     // Send out of band
-    later(() => {
-      if (this.isDestroying) {
-        return;
-      }
-      this.releaseCurrentObjects();
-      let tree = this.viewTree();
-      if (tree) {
-        this.sendMessage('viewTree', { tree });
-      }
-    }, 50);
+    throttle(this, this.send, 500, false);
```

Comment:
Is the idea of this throttle to cut down on the number of renders?

Reply:
yes, though I want to reduce the lag to 300, which feels a bit better

Reply:
mmm, right, this is because i set it to call the function at the end of 500ms. to send it out of band.
but I can  actually do later(() => throttle(this, this.send, 300), 50);
so, we get a fast new render, but if there are too many rerendes it will delay it by 300ms to not kill ember-inspector

Reply:
Could we try `throttle(this, this.send, 100, true);`? To allow it to be called immediately, then throttle to 100ms?

Reply:
Sure, we can try 300 then. Perhaps we should build in some kind of assertion here that if the app is going crazy with a ton of renders, that we warn you to look into the app?